floyd/cli/run.py

```diff
@@ -53,8 +53,8 @@
 # or <namespace>/[projects|datasets]/<dataset_or_project_name>:<mount_dir>
 # or <namespace>/[projects|datasets]/<dataset_or_project_name>/<version>
 # or <namespace>/[projects|datasets]/<dataset_or_project_name>/<version>:<mount_dir>
-DATANAME_PATTERN = '(%s+/datasets/)?%s+(/[0-9]+)?(:/?%s+/?)?' % (ALLOWED_CHARSET, ALLOWED_CHARSET, ALLOWED_CHARSET)
-PRJFILE_PATTERN = '(%s+/projects/)?%s+(/[0-9]+(/output)?)?(:/?%s+/?)?' % (ALLOWED_CHARSET, ALLOWED_CHARSET, ALLOWED_CHARSET)
+DATANAME_PATTERN = '(%s+/datasets/)?%s+(/[0-9]+)?:/?%s+/?' % (ALLOWED_CHARSET, ALLOWED_CHARSET, ALLOWED_CHARSET)
+PRJFILE_PATTERN = '(%s+/projects/)?%s+(/[0-9]+(/output)?)?:/?%s+/?' % (ALLOWED_CHARSET, ALLOWED_CHARSET, ALLOWED_CHARSET)
```

Comment:
Can you rename this as `PROJECT_OUTPUT_PATTERN`? I noticed this was in the previous commit but can you fix it? Thx!

Reply:
Sure!